pkg/cmd/api/api.go

```diff
@@ -324,10 +326,10 @@ func apiRun(opts *ApiOptions) error {
 		}
 	}
 
-	return nil
+	return template.End()
```

Comment:
When the user hasn't passed any `--template` in this invocation, `End()` is essentially a no-op, right?

Reply:
Yes, and does still work for templates without a table row, such as this one I configured for testing:
```yaml
labels2: |                                                                                                                                                                  
    api /repos/:owner/:repo/labels --template '{{range .}}{{autocolor "green" .name}} {{printf "#%v" .color}} {{autocolor "gray+h" .description}}{{"\n"}}{{end}}' --paginate
```

Still, I can look into conditionally eliding it.

Reply:
I remember now: since the template function was converted to a module, this was meant to be a finalizer and should no op if there's nothing to do. Once I lazy init the table printer, the function will do a more representative job of that.